activemq-karaf/src/main/resources/features-core.xml

```diff
@@ -20,9 +20,12 @@
 
 <features name="activemq-core-${project.version}">
 
+    <repository>mvn:org.apache.karaf.features/spring-legacy/[4,5)/xml/features</repository>
```

Comment:
Hi @jbonofre,
this line causes my karaf always to resolve featurea from karaf 4.2.2 SNAPSHOT if using 5.15.7 ActiveMQ feature. Might be a bug in the karaf feature resolver?
Regards,
Benjamin